daft/runners/partitioning.py

```diff
@@ -271,6 +271,92 @@ def wait(self) -> None:
         raise NotImplementedError()
 
 
+class LocalPartitionSet(PartitionSet[MicroPartition]):
```

Comment:
Is this a pure movement of the class from pyrunner.py? Why did we decide to move it?

Reply:
It was being imported by dataframe.py and ray_runner.py which was causing some circular import issues. I figured since it was being used outside of the pyrunner I would move it out to simplify the dependency tree.